cadquery/vis.py

```diff
@@ -209,6 +233,10 @@ def show(
     renderer.AddActor(pts)
     renderer.AddActor(axs)
 
+    # add other vtk actors
+    for p in props:
+        renderer.AddActor(p)
+
     # initialize and set size
     inter.Initialize()
     win.SetSize(*win.GetScreenSize())
```

Comment:
Would you be open to changing the default window size?  The default window size is too large IMO.  As is, I must manually resize the viewer window each time it opens if I want to have a code editor and CQ viewer window open on the same screen.

I wouldn't need to fiddle with viewer window as much if it was reduced in size, say something like this:
```py
win.SetSize(tuple(int(v*0.4) for v in win.GetScreenSize()))
```

Reply:
OK, I changed to w,h /2.

Reply:
Great, thanks @adam-urbanczyk !